tests/conftest.py

```diff
@@ -699,6 +706,155 @@ def _launch_image(image: DockerImage, timeout: float = 2.0, max_attempts: int =
             container.remove()
 
 
+def _pre_launch(image: DockerImage) -> None:
```

Comment:
You understand we went through a massive exercise to remove the code here so we could run it isolated in packages? It makes no sense placing it here.